### PR TITLE
[IOS-1909] Updating to ver3

### PR DESCRIPTION
**Pull request by AmitaiB**
Incorporates @karimMourra 's comments.

This also adds an `IDEWorkspaceChecks.plist` file.

> Xcode 9.3 adds a new IDEWorkspaceChecks.plist file to a workspace's shared data, to store the state of necessary workspace checks. Committing this file to source control will prevent unnecessary rerunning of those checks for each user opening the workspace. (37293167)

--Apple: [Xcode Release Notes](https://developer.apple.com/library/archive/releasenotes/DeveloperTools/RN-Xcode/Chapters/Introduction.html)

cf. SERV-1000

### CODE REVIEW

File: JWBestPracticeApps/JWBestPracticeApps.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1593,43 +1574,16 @@
...
+				"${PODS_PODFILE_DIR_PATH}/Podfile.lock",
+				"${PODS_ROOT}/Manifest.lock",
 			);
 			name = "[CP] Check Pods Manifest.lock";
 			outputPaths = (
+				"$(DERIVED_FILE_DIR)/Pods-JWRemoteCastController-checkManifestLockResult.txt",
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
```
Comment (karimMourra) [inline JWBestPracticeApps/JWBestPracticeApps.xcodeproj/project.pbxproj]: @AmitaiB  what are all these changes ?

Comment (AmitaiB) [reply JWBestPracticeApps/JWBestPracticeApps.xcodeproj/project.pbxproj]: The project was created with CocoaPods 1.0.1, and I ran `pod update` using CocoaPods 1.5.3. I presume that they updated their own boilerplate.

File: JWBestPracticeApps/Podfile.lock [hunk 2]
```diff
@@ -5,16 +5,21 @@ PODS:
   - google-cast-sdk/Assets (2.10.0):
     - google-cast-sdk/Core
   - google-cast-sdk/Core (2.10.0)
-  - JWPlayer-SDK (2.5.4)
+  - JWPlayer-SDK (3.2.3)
 
 DEPENDENCIES:
   - google-cast-sdk (<= 2.10.0.4070)
-  - JWPlayer-SDK (~> 2.0)
+  - JWPlayer-SDK (~> 3.2)
```
Comment (karimMourra) [inline JWBestPracticeApps/Podfile.lock]: you may need to run pod update; this should have been automatically changed to 3.0

**Comment by okulagin** [comment]
@AmitaiB would you address Karim's comments please?

File: JWBestPracticeApps/Podfile.lock (hunk 2 above)
Comment (AmitaiB) [reply JWBestPracticeApps/Podfile.lock]: The [Podfile.lock](https://guides.cocoapods.org/using/using-cocoapods.html#what-is-podfilelock) file was changed _by_ pod update, not by me manually. Does this address your comment?

Comment (karimMourra) [reply JWBestPracticeApps/Podfile.lock]: right, that' s why i'm saying you need to run pod update in order for this to update automatically.

**Comment by AmitaiB** [comment]
This PR is made obsolete by IOS-1944, closing.

**Pull request closed without merging**